koko_gym/envs/koko_reacher.py

```diff
@@ -34,11 +59,29 @@ def reset_model(self):
         return self._get_obs()
 
     def _get_obs(self):
-        theta = self.model.data.qpos.flat[:2]
-        return np.concatenate([
-            np.cos(theta),
-            np.sin(theta),
-            self.model.data.qpos.flat[2:],
-            self.model.data.qvel.flat[:2],
-            self.get_body_com("fingertip") - self.get_body_com("target")
-        ])
+        return np.concatenate([self.sim.data.qpos,self.sim.data.qvel,self.get_body_com("robotleftfingertip") - self.get_body_com("target")])
+
+    def random_policy(self):
```

Comment:
Minor nitpick -- can we move this policy outside of the environment?

Reply:
No problem with deleting the random policy from the environment. Yeah, that should not be the environment function.

Reply:
Will you change it on your side?

Reply:
Yep, I can do it & merge

Reply:
Okay! Great, then I will resolve this conversation.